contributing/diversity/governance.rst

```diff
@@ -72,6 +72,23 @@ For an actionable item to pass, approval from greater than 50% of the voting key
 players is required. Use or management of finances/donations require at least a
 two-thirds majority to pass.
 
+For transparency and ease-of-understanding, this means only the following
+combinations of votes will result in an actionable item passing:
+
++-----+---------+---------+------------------------------+
+| For | Against | Abstain | Relative Approval Percentage |
```

Comment:
Anyone have any opinions on whether the final column, _Relative Approval Percentage_, is necessary or not?

Reply:
I think we can skip it.